test/TestCases/Case02/case.json

```diff
@@ -6,8 +6,8 @@
             "--output-js-resolver-file", "./test/TestCases/Case02/output/output.resolver.graphql.cjs",
             "--output-folder-path", "./test/TestCases/Case02/output",
             "--output-no-lambda-zip"],
-    "host": "<your-airroutes-db-host>",
-   "port": "<your-airroutes-db-port>"
+    "host": "g-hgb6zpdbh5.us-west-2.neptune-graph.amazonaws.com",
```

Comment:
Endpoints should never be committed; please remove and clean up the GitHub History.